x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/chart_switch.tsx

```diff
@@ -54,6 +51,18 @@ interface Props {
   >;
 }
 
+interface SelectableEntry {
```

Comment:
nit: You can replace this by `type SelectableEntry = EuiSelectableOption<{ value: string }>`.

There's one small issue with the type of the array returned in 272 because `concat` is not playing nice, but you can cast it using `as SelectableEntry[]`

Reply:
Yes, that was the main reason to redeclare it from scratch. I'll see if it's possible to fix it without force casting it. In case it's not possible, I'll force cast 😅